inst/shiny-examples/SCD-effect-sizes/code-chunks/dat-clean.R

```diff
@@ -0,0 +1,10 @@
+# clean data
+library(dplyr)
+
+dat <-
+  dat %>%
+  group_by({user_grouping}) %>%
+  arrange({user_grouping}, {user_session_number}) %>%
```

Comment:
Skip the `arrange()` step and use the `session` argument of `calc_phase_pairs()`.